backend/account_v2/subscription_loader.py

```diff
@@ -75,3 +76,32 @@ def load_plugins() -> list[Any]:
         logger.info("No subscription plugins found.")
 
     return subscription_plugins
+
+
+def validate_etl_run(org_id: str) -> bool:
```

Comment:
@muhammad-ali-e can this be moved to the cloud repo?

Reply:
It also exists in V1, which is why I copied it here. I’m not fully aware of the complete context, but we can move it to the appropriate locations once the V2 migration is complete.